.github/scripts/package_and_deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+
+
+ddprof_deploy_folder=$1
+commit_sha=$2
+commit_author=$3
+
+
+profiler_version=v1.1.1_$(date -u +%G%m%d%H%M%S)
```

Comment:
will do thank

Reply:
IIRC we have settled on having the same version. So should this be bumped when we bump the tracer version?

Reply:
🤔 not sure about that, meaning for the combined tracer/profiler package yes, but this one is profiler only.
Once we have a combined package yes (btw, once we have a combine package, this script will be removed)

Reply:
> Once we have a combined package

What do you mean by "package" here? 🤔 We're already shipping the combined MSI right?

Reply:
By package, I meant a lot of things. Let me clarify.
Today, we have the MSI, but our Rel.Env. relies on the zip file created by this script. 
In this zip, there is only the profiler. No tracer, nor the native loader/dispatcher (since we were in 2 different repos)